transport.go

```diff
@@ -98,9 +103,39 @@ func addWebhookRequestDecoder(config transportConfig) kithttp.DecodeRequestFunc
 
 		wv.setWebhookDefaults(&webhook, r.RemoteAddr)
 
+		auth, _ := bascule.FromContext(c)
+		var partners []string
+
+		switch auth.Token.Type() {
+		case "Basic":
+			var configurableString string
+			authHeader := r.Header[configurableString]
+
+			for _, value := range authHeader {
+				fields := strings.Split(value, ",")
+				for i := 0; i < len(fields); i++ {
+					fields[i] = strings.TrimSpace(fields[i])
+				}
+				partners = append(partners, fields...)
+			}
+		case "jwt":
+			authToken := auth.Token
+			whatdoesthisinterfacemean, _ := bascule.GetNestedAttribute(authToken.Attributes(), basculechecks.PartnerKeys()...)
+			vals, err := cast.ToStringSliceE(whatdoesthisinterfacemean)
+			if err != nil {
+				return nil, errGettingPartnerIDs
```

Comment:
```suggestion
				return nil, fmt.Errorf("%w: %v", errGettingPartnerIDs, err)
```
Usually we don't want to lose any error information.